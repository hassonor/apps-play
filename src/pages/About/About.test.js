import React from "react";
import {shallow} from "enzyme";
import About from "./About";

it("expect to render About Page", () => {
    expect(shallow(<About/>)).toMatchSnapshot();
});