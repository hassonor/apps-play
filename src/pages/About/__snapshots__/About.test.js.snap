// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render About Page 1`] = `
<Fragment>
  <h1
    className="text-6xl mb-4"
  >
    IronSource Apps Services
  </h1>
  <p
    className="mb-4 text-2xl font-light"
  >
    A React app to search IronSource Apps Services.
  </p>
  <p
    className="text-lg text-gray-400"
  >
    Version 
    <span
      className="text-white"
    >
      1.0.0
    </span>
  </p>
  <p
    className="text-lg text-gray-400"
  >
    Layout By:
    <span
      className="text-white"
    >
      Or Hasson
    </span>
  </p>
</Fragment>
`;
